src/geotop/timer.h

```diff
@@ -0,0 +1,52 @@
+//
+// Created by elisa on 29/10/18.
+//
+#ifndef GEOTOP_TIMER_H
+#define GEOTOP_TIMER_H
+
+#include <iostream>
+#include <chrono>
+#include <map>
+
+using namespace std::chrono;
+
+class Timer {
+public:
+    std::map<std::string, std::pair<int,duration<double>>> times;
+    ~ Timer() {
+        for (const auto& x:times){
+            std::cout << " --------------------------------------------------- " << std::endl;
+            std::cout << x.first << "       " /** function name **/

```

Comment:
as a quick fix here I would use one or more `\t` instead of the spaces to have a bit of alignment